offerItemImport/offerITemImportHandler.go

```diff
@@ -32,9 +32,10 @@ func NewOfferItemImportHandler(deps Deps) OfferItemImportHandlerInterface {
 }
 
 func (oi *OfferItemImportHandler) Run() {
+	log.Println("_________________________________")
 	files := adapters.GetFiles(oi.sourcePath)
 	if len(files) == 0 {
-		log.Printf("Offer Item Import failed: please, put file with offers into %v", oi.sourcePath)
+		log.Printf("Import offeritems failed: please, put file with offers into %v", oi.sourcePath)
```

Comment:
This doesn't look like correct English phrase. 
Is offeritems more readable then Offer Item?